relay-log/build.rs

```diff
@@ -17,9 +17,8 @@ fn emit_release_var() -> Result<(), io::Error> {
         ));
     }
 
-    let version = std::env::var("CARGO_PKG_VERSION").unwrap();
     let revision = String::from_utf8_lossy(&cmd.stdout);
-    println!("cargo:rustc-env=RELAY_RELEASE=relay@{version}+{revision}");
+    println!("cargo:rustc-env=RELAY_RELEASE=relay@{revision}");
```

Comment:
Why is this change needed?

Reply:
read the PR description - it's part of sentry release names

Reply:
Please revert this, we explicitly want this version number in our builds.

Reply:
Hmm... i'll have to think of a way gocd can get the semver then. 

How about this? Have an additional RELAY_SHA that'll be passed to sentry-sdk for release names. RELAY_RELEASE can stay.

Reply:
The point is to have both the version number and the SHA in the sentry-sdk too. I understand the limitation of not running docker-in-docker, could we consider to use the devinfra for this? I'm kinda uneasy about enforcing a specific versioning schema (i.e. change the application) just because the CI pipeline can't obtain the actual version number. 

Worst case, we just don't create the release here and let sentry auto-create it. It would be great to have it created eagerly, but I'd choose names over eager creation.

Reply:
As discussed in Slack I'll switch to writing the release name to the deployment bucket. It's still way better than running d-in-d and pulling an entire image.